### PR TITLE
Update minor release guide after 713

### PR DESCRIPTION
**Pull request by mboskamp**

### CODE REVIEW

**Comment by mboskamp** [comment]
@yanavasileva, please don't merge the branch after the review. I want to get more feedback from INFRA once everything is OK from our side.

**Comment by yanavasileva** [comment]
Switch to Slack announcement:
* https://github.com/camunda/camunda-bpm-dev-docs/blob/update-minor-release-guide-after-713/releases/Performing-a-Minor-Release.md#5-send-an-email-to-camundabpmcamundacom

**Comment by yanavasileva** [comment]
I asked Micha if he is ok to the change in the patch release guide.

File: releases/Performing-a-Minor-Release.md [hunk 1]
```diff
@@ -250,13 +263,59 @@ When releasing a secondary project, other secondary projects might have cross-re
...
+### Create maintenance branches
+
+Trigger the creation of the maintenance branches with [this job](https://release.cambpm.camunda.cloud/view/all/job/maintenance/job/RELEASE-create-camunda-bpm-maintenance-branch/).
+
+Check that new branches were created in the following repositories:
+* https://github.com/camunda/camunda-bpm-platform-maintenance
+* https://github.com/camunda/camunda-bpm-platform-ee-maintenance
+* https://github.com/camunda/docker-camunda-bpm-platform
+
+### Create new minor version branches
```
[the remark this answers is not in the document]
Comment (mboskamp) [reply releases/Performing-a-Minor-Release.md:300]: As we already discussed, in this step we will execute a script that creates branches and jobs not only for the docs, but also for other repos. It is important to execute it before the stating of the jobs as the job to stage them is created here. As discussed, I will move this step right before `Stage new version of the docs`.

**Comment by mboskamp** [comment]
@yanavasileva, I implemented your feedback as we discussed. I also merged a change regarding updating screenshots already as it was missing in the table of contents in the original commit. The link will work after this is merged to master.
If you are ok with the guide, let me know. I want to get feedback from Infra before merging.

Thanks,
Miklas

File: releases/Performing-a-Minor-Release.md [hunk 2]
```diff
@@ -354,10 +327,53 @@ If everything is satisfying with the release, send a message to `#cambpm-announc
...
+```
+
+Check that new branches were created in the following repositories:
+* https://github.com/camunda/camunda-bpm-platform-maintenance
+* https://github.com/camunda/camunda-bpm-platform-ee-maintenance
+* https://github.com/camunda/docker-camunda-bpm-platform
+* https://github.com/camunda/camunda-bpm-webapp
+* https://github.com/camunda/camunda-webapp-translations
+
+### Create new minor version branches
```
Comment (yanavasileva) [inline releases/Performing-a-Minor-Release.md:358]: I think it should be new minor version `jobs` instead of `branches`, right?

**Comment by yanavasileva** [comment]
@mboskamp I added a commit with some changes (if you don't agree, please feel free to revert it), other than that you have a green light from my side. As Daniel was faster than us, there is a conflict due to the manager role change.

**Comment by mboskamp** [comment]
@AndreaGiardini, as discussed, please have a look on the INFRA related changes in our minor release guide and let me know if you would like to change something.

**Comment by AndreaGiardini** [comment]
Me and @aabouzaid will have a look at it early next week, thank you for pinging us :)

**Pull request merged**